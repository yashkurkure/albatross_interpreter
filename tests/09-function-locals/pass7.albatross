fun f1 int () {
  var i int := f1();
  return 0;
}

f1();
return 0;
