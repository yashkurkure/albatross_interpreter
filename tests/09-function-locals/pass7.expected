Function declared "f1" returns int
	Local variable "i" type int position 0
Function called "f1" returns int
Function called "f1" returns int
